server/controllers/admin/locations.js

```diff
@@ -549,3 +549,73 @@ exports.delete.village = (req, res, next) => {
     res.sendStatus(204);
   }).catch(next);
 };
+
+exports.merge = (req, res, next) => {
+  const { selected, other, locationStatus } = req.body;
+
+  const transaction = db.transaction();
+
+  if (locationStatus === 'country') {
+    const replaceCountryInProvince = `
+      UPDATE province SET country_uuid = ? WHERE country_uuid = ?;`;
+
+    const removeOtherCountry = `
+      DELETE FROM country WHERE uuid IN (?);`;
```

Comment:
Are you possibly deleting more than one country?   If we only delete one, perhaps this should use 'uuid = ?' rather than IN?